api/nodecontrol/auction-response.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "io.nats.nex.v2.auction_response",
+  "title": "AuctionResponse",
+  "type": "object",
+  "properties": {
+    "node_id": {
+      "type": "string",
+      "description": "The unique identifier of the node"
+    },
+    "nexus": {
+      "type": "string",
+      "description": "The name of the nexus - if assigned"
```

Comment:
This will always be here. If the nexus name isn't assigned it gets the default name `nexus` (I think that's the default)